lldb/include/lldb/Target/Process.h

```diff
@@ -1104,9 +1104,15 @@ class Process : public std::enable_shared_from_this<Process>,
   /// \see Thread:Resume()
   /// \see Thread:Step()
   /// \see Thread:Suspend()
-  virtual Status DoResume() {
-    return Status::FromErrorStringWithFormatv(
-        "error: {0} does not support resuming processes", GetPluginName());
+  virtual Status DoResume(lldb::RunDirection direction) {
+    if (direction == lldb::RunDirection::eRunForward) {
+      return Status::FromErrorStringWithFormatv(
+          "error: {0} does not support resuming processes", GetPluginName());
+    } else {
+      return Status::FromErrorStringWithFormatv(
+          "error: {0} does not support reverse execution of processes",
+          GetPluginName());
+    }
```

Comment:
```suggestion
    return Status::FromErrorStringWithFormatv(
          "error: {0} does not support reverse execution of processes",
          GetPluginName());
```

https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Reply:
Done. I fixed this everywhere in the PR.